native_client/swift/stt_ios_test/SpeechRecognitionImpl.swift

```diff
@@ -8,149 +8,72 @@
 //  Copyright © 2021 Coqui GmbH
 import Foundation
 import AVFoundation
-import AudioToolbox
 import Accelerate
 
 import stt_ios
 
-struct FillComplexInputParm {
-    var source: UnsafeMutablePointer<Int8>
-    var sourceSize: UInt32
-};
-
-class SpeechRecognitionImpl : NSObject, AVCaptureAudioDataOutputSampleBufferDelegate {
+class SpeechRecognitionImpl {
     private var model: STTModel
     private var stream: STTStream?
 
-    private var captureSession = AVCaptureSession()
+    private let modelFeedInterval = 0.1
```

Comment:
The point of my previous review was to decouple `feedAudioContent` from `intermediateDecode`. Here, you put both inside a timer callback, but they're still being called in lockstep. Every time you feed, you decode. On top of that, decoding every 0.1s is quite wasteful, the results can only change after 16 timesteps are collected (by default, can be tweaked during model export).